modules/50_damages/TC/postsolve.gms

```diff
@@ -9,7 +9,8 @@
 pm_damageGrowthRateIso(tall,iso) = 0;
 
 *the current damage function is estimated for "baseline" TC effect plus climate change effect - need to subtract the baseline effect as in Burke by doing delta(T)-delta(T0)
-pm_damageGrowthRateIso(tall,isoTC)$(tall.val ge 2000 and tall.val le 2300) = p50_damageFuncCoefTC1(isoTC) * (pm_globalMeanTemperatureZeroed1900(tall)-pm_globalMeanTemperatureZeroed1900("2000")) + p50_damageFuncCoefTC2(isoTC) * (pm_globalMeanTemperatureZeroed1900(tall)**2-pm_globalMeanTemperatureZeroed1900("2000")**2); 
```

Comment:
I don't really understand why you changed that

Reply:
I have to go back to the TC data, I remember there is no squared term in the final version.